src/compiler/transformers/module/module.ts

```diff
@@ -1813,21 +1839,61 @@ namespace ts {
         }
     }
 
+    export const createBindingHelper: UnscopedEmitHelper = {
+        name: "typescript:commonjscreatebinding",
+        importName: "__createBinding",
+        scoped: false,
+        priority: 1,
+        text: `
+var __createBinding = (this && this.__createBinding) || (Object.create ? (function(o, m, k, k2) {

```

Comment:
This will also need to be in tslib.

Reply:
Yep.

Reply:
https://github.com/microsoft/tslib/pull/89